language/move-prover/boogie-backend/src/boogie_helpers.rs

```diff
@@ -329,6 +329,24 @@ pub fn boogie_address_blob(_options: &BoogieOptions, val: &[BigUint]) -> String
     }
 }
 
+pub fn boogie_constant_blob(_options: &BoogieOptions, val: &[Constant]) -> String {
+    let args = val.iter().map(|v| format!("{}", *v)).collect_vec();
```

Comment:
This doesn't seem quite right to me.., I don't think any `Constant` can be `format!()` into boogie-recognizable text.

Perhaps a better solution is to have a helper function:
```rust
pub fn boogie_constant(options: &BoogieOptions, val: &Constant) -> String { .. }
```

And then call this `boogie_constant` inside the `.map()`